<html><head></head><body><pre>--- /home/wxwindows/public_html/cvsdocs/wxWidgets/docs/latex/wx/listattr.tex	2005-06-26 15:51:29.000000000 +0200
+++ docs/latex/wx/listattr.tex	2005-08-19 15:40:51.000000000 +0200
@@ -1,4 +1,4 @@
-\section{\class{wxListItemAttr}}\label{wxListItemAttr}
+\section{\class{wxListItemAttr}}\label{wxlistitemattr}
 
 Represents the attributes (color, font, \ldots) of a
 \helpref{wxListCtrl}{wxlistctrl} \helpref{wxListItem}{wxlistctrl}.
@@ -39,6 +39,7 @@
 
 
 \membersection{wxListItemAttr::GetTextColour}\label{wxlistitemattrgettextcolour}
+
 \constfunc{const wxColour\&amp;}{GetTextColour}{\void}
 
 Returns the currently set text color.
@@ -80,6 +81,7 @@
 
 
 \membersection{wxListItemAttr::SetTextColour}\label{wxlistitemattrsettextcolour}
+
 \func{void}{SetTextColour}{\param{const wxColour\&amp; }{colour}}
 
 Sets a new text color.
</pre></body></html>
 	  	 
